### DISCUSSION TITLE
How can I get the next time a job is scheduled

### DISCUSSION THREAD

**Discussion by IgnisDa** [dkron-io/dkron#1270] [Q&A] [answered]
Hello! As far as I can see https://dkron.io/api/#operation/showJobByName endpoint does not have any property that will tell me when a job is scheduled to run next. Is it possible or do I have to calculate it manually?

**Comment by IgnisDa** [comment, marked as answer]
Looks like there is `.next` property in the actual response, however it is not visible in the API docs. Are the docs not updated to the latest schema?

**Reply by IgnisDa** [reply]
It is not present in the schema too https://github.com/distribworks/dkron/blob/76876e1b2f9e39fcffc20da5c440ccd23ce7fb10/website/static/openapi/openapi.yaml#L393-L508

**Reply by yvanoers** [reply]
Apparently we've missed this in the documentation. We should add it as a readonly property.

**Comment by vcastellm** [comment]
Added to docs
